src/view/screens/ProfileFeed.tsx

```diff
@@ -221,18 +223,19 @@ export function ProfileFeedScreenInner({
       logger.error('Failed up update feeds', {message: err})
     }
   }, [
-    feedInfo,
+    isHapticsDisabled,
     isSaved,
-    saveFeed,
     removeFeed,
-    resetSaveFeed,
+    feedInfo.uri,
```

Comment:
Unintended change? I suppose it doesn't matter but I'm not sure

Reply:
Linter probably picked them up when I added to the deps. Is probably fine, I'm fine with reverting the lint changes though if we want.

Reply:
Same to the others below.